src/main/java/seedu/ichifund/model/loan/Loan.java

```diff
@@ -0,0 +1,143 @@
+package seedu.ichifund.model.loan;
+
+import static seedu.ichifund.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+
+import seedu.ichifund.model.amount.Amount;
+
+import seedu.ichifund.model.date.Date;
+import seedu.ichifund.model.person.Name;
```

Comment:
Hi, can you change this import to `loan.person` instead?

Reply:
moved name to loan